lib/utils.js

```diff
@@ -23,3 +25,10 @@ exports.getImageMimeType = function getImageMimeType (imagePath) {
       return undefined
   }
 }
+
+exports.isRevealTheme = function isRevealTheme (theme) {
+  if (fs.existsSync(path.join(process.cwd(), 'public', 'build', 'reveal.js', 'css', 'theme', theme + '.css'))) {
```

Comment:
`__dirname` would give `/home/user/hackmd/lib/` happy to switch  to it itll just have to be `__dirname, '..'`